README.md

```diff
@@ -96,6 +96,7 @@ You can access these pages on your computer using one of the following clients:
     [App Store](https://itunes.apple.com/us/app/tldt-pages/id1071725095?ls=1&mt=8)
 - [Haskell client](https://github.com/psibi/tldr-hs):
   `stack install tldr`
+  or `apt-get install tldr` on Debian
```

Comment:
Does this work on Ubuntu too? (I guess it does)
It should be noted in some way. :p

Reply:
@zdroid, I search in [Ubuntu Packages](https://packages.ubuntu.com/search?keywords=Tldr&searchon=names&suite=eoan&section=all) and yes, in both correspond with the Haskell client. As soon as possible I will note this and the [one for the Python client](https://packages.ubuntu.com/eoan/tldr-py), although it seems a packaged version of lord63/tldr.py, which its last commit was at November 27, 2017.

I will change the title of the PR too.

Reply:
@zdroid, look at the new changes in which I added the improvement you suggested.

Reply:
Ubuntu is based on Debian, so it's perhaps best to mention the base distribution here - since there are lots of different Debian-based distributions - we wouldn't have space to mention them all :P